### DISCUSSION TITLE
<Conversion error in Deployment of Website>

### DISCUSSION THREAD

**Discussion by jackstephwhite** [community/community#47909] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I recieved this error. I tried running with the root branch as well as /docs branch. Not sure what to do. New to web development of any kind.

Conversion error: Jekyll::Converters::Scss encountered an error while converting 'assets/css/style.scss':
                    No such file or directory @ dir_chdir - /github/workspace/docs

**Comment by airtower-luna** [comment]
Can you link the actual repository? Without that it's hard to answer, though it looks like the deployment path (not branch) might be wrong.
